modules/telemetry/telemetrymodule.cpp

```diff
@@ -0,0 +1,314 @@
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/telemetry/telemetrymodule.h>
+
+#include <modules/telemetry/include/general/anglemodetelemetry.h>
+#include <modules/telemetry/include/general/cameratelemetry.h>
+#include <modules/telemetry/include/general/focustelemetry.h>
+#include <modules/telemetry/include/general/nodestelemetry.h>
+#include <modules/telemetry/include/general/timetelemetry.h>
+#include <modules/telemetry/include/specific/planetscomparesonification.h>
+#include <modules/telemetry/include/specific/planetsoverviewsonification.h>
+#include <modules/telemetry/include/specific/planetssonification.h>
+#include <ghoul/logging/logmanager.h>
+#include <openspace/camera/camera.h>
+#include <openspace/documentation/documentation.h>
+#include <openspace/engine/globals.h>
+#include <openspace/engine/globalscallbacks.h>
+#include <openspace/engine/windowdelegate.h>
+#include <openspace/rendering/renderengine.h>
+#include <openspace/scene/scene.h>
+
+namespace {
+    constexpr std::string_view _loggerCat = "TelemetryModule";
+
+    // The default OSC reciver is SuperCollider with these default values.
+    // However, the user can define any reciver in the openspace.cfg file as the
+    // ModuleConfiguration for the Telemetry module.
+    constexpr std::string_view DefaultSuperColliderIp = "127.0.0.1";
+    constexpr int DefaultSuperColliderPort = 57120;
+
+    constexpr openspace::properties::Property::PropertyInfo EnabledInfo = {
+        "Enabled",
+        "Enabled",
+        "Enable or disable all gathering of telemetry information"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo IpAddressInfo = {
+        "IpAddress",
+        "Ip address",
+        "The network ip address that the telemetry osc messages is sent to"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo PortInfo = {
+        "Port",
+        "Port",
+        "The network port that the telemetry osc messages is sent to"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo AngleCalculationModeInfo = {
+        "AngleCalculationMode",
+        "Angle Calculation Mode",
+        "This setting changes the method to calculate any angles in the telemetries. "
+        "The Horizontal mode, generally works well for flat displays or forward facing "
+        "immersive envierments. The Circular mode, generally works well for centered "
+        "fisheye displays or omnidirectional immersive environments"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo IncludeElevationAngleInfo = {
+        "IncludeElevationAngle",
+        "Include Elevation Angle",
+        "This setting determines if an additional elevation angle should be calculated "
```

Comment:
What is the elevation angle, though?  I guess this is what your docs page will explain